test/controllers/check.your.answers.controller.spec.ts

```diff
@@ -1177,29 +1167,8 @@ describe("POST tests", () => {
     expect(resp.text).toEqual(`${FOUND_REDIRECT_TO} ${PAYMENT_LINK_JOURNEY}`);
   });
 
-  test(`redirect the ${CONFIRMATION_PAGE} page after a successful post from ${CHECK_YOUR_ANSWERS_PAGE} page`, async () => {
```

Comment:
This test still sounds relevant but I guess it was specific to 'save and resume' and the scenario is covered by another test? Same for a number of tests being removed below...

Reply:
I'm not sure, i'll look into it.

Reply:
they were basically repeats of existing tests, once the save and resume flag branch of code was made default in the controller the tests became almost identical to those just above. The redirect page difference is only due to the individual test overriding the redirect page name returned from the mock.
I've added some extra 'expects' in to beef up a couple of these tests in this area.